lib/plugins/aws/invoke/index.js

```diff
@@ -28,6 +28,16 @@ class AwsInvoke {
     // validate function exists in service
     this.options.functionObj = this.serverless.service.getFunction(this.options.function);
 
+    // if data is provided via CLI, sanitize it
+    if (typeof this.options.data === 'string') {
+      try {
+        this.options.data = JSON.parse(this.options.data);
```

Comment:
Here is a problem: what if I want the data to come to Lambda as string? 

In number of cases, including default "PROXY", the body is passed to handler's `event` as string, and Lambda handler needs to convert the string to JSON. 

Please have an option to skip this conversion.